source/Calamari.AzureWebApp/Calamari.AzureWebApp.csproj

```diff
@@ -22,6 +22,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Calamari.AzureScripting\Calamari.AzureScripting.csproj" />
+    <ProjectReference Include="..\Calamari.Azure\Calamari.Azure.csproj" />
```

Comment:
If we move to using the Authentication from Calamari.CloudAccounts I think this can be removed.

Reply:
It still need for AzureClient extension method